riscv/decode.h

```diff
@@ -135,9 +135,9 @@ class insn_t
   uint64_t v_vta() { return x(26, 1); }
   uint64_t v_vma() { return x(27, 1); }
   uint64_t v_mew() { return x(28, 1); }
-  uint64_t v_simm12() {return xs(20,12);}
-  uint64_t v_simm11() {return xs(20,11);} 
-  uint64_t v_s_simm11() { return x(7, 5) + (xs(25, 6) << 5); }
+  uint64_t v_simm12() {return xs(20,12);} //for inst vls12 series
```

Comment:
keep comment style as below
// for inst vls12 series
space is needed after `//`